packages/timeline-state-resolver/src/__mocks__/net.ts

```diff
@@ -33,13 +33,12 @@ export class Socket extends EventEmitter {
 	// this.emit('close')
 	// this.emit('end')
 
-	public connect(port, host, cb) {
+	public connect(port: number, host: string) {
```

Comment:
A connectListener is part of the [socket.connect API](https://nodejs.org/docs/latest-v14.x/api/net.html#net_socket_connect): I think it would be good to keep it. Maybe then the Node 16 tests wouldn't fail? IDK.

Reply:
cb re-added as optional param. I see the docs you point at says host is optional too, will change that too.